### ISSUE TITLE
Scroll working badly.

### ISSUE THREAD

**Issue by JuliGame** [ATLauncher/ATLauncher#866]
### What happened?

Scroll only works when mouse cursor is on background. Using latest aur (arch) release
Video: 

https://github.com/ATLauncher/ATLauncher/assets/60196802/52974ee0-4f88-4f7c-b20a-a927710cf41b

### Operating System

Linux

### Is this actually a launcher bug?

- [X] I am sure this is a bug with the launcher and not an issue with running Minecraft

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by RyanTheAllmighty** [comment]
Yes this has been an issue for a long time and is on all platforms. Technically you're scrolling the description area which is a scroll field, so trying to scroll that will ONLY scroll that description field, not the container. It's technically not a bug, but it's not ideal and should be fixed anyway

**Comment by WaveSmashies** [comment]
This has been a problem for a long time now. Can we get this fixed please? Lol. Also it's easier to scroll if you just hover the mouse over the image for the instance. (though that's not an excuse not to fix this.)

**Comment by ellet0** [comment]
While this issue probably can be fixed, the general experience of Java Swing is not that smooth on most modern software and hardware, I'm surprised it's still working, to get more smoother experience we might need a better graphics engine like Skia, Compose desktop or Java FX is more modern and should be work with the existing source code without breaking anything or using a different framework and build the project from scratch.
